acceptance-tests/tests/src/test/java/org/hyperledger/besu/tests/web3j/privacy/OnChainPrivacyAcceptanceTest.java

```diff
@@ -103,6 +106,96 @@ public void canAddParticipantToGroup() {
     charlie.verify(privateTransactionVerifier.existingPrivateTransactionReceipt(commitmentHash));
   }
 
+  @Test
+  public void aliceCanAddRemoveAddBob() {
```

Comment:
I'd like to see a more descriptive name in this test. Something like `memberCanBeAddedAfterBeingRemoved` or something similar.

Reply:
done